js/index.js

```diff
@@ -0,0 +1,84 @@
+let books = null;
+function sortBooks() {
+  books.sort((bookA, bookB) => {
+    if (bookA.id < bookB.id) {
+      return 1;
+    }
+    if (bookA.id > bookB.id) {
+      return -1;
+    }
+    return 0;
+  });
+}
+function displayBooks() {
+  sortBooks();
+  const section = document.getElementById('collection');
+  const list = document.createElement('ul');
+  list.id = 'list';
+  books.forEach((book) => {
+    const { title } = book;
+    const { id } = book;
+    const liId = `li${title}`;
+    const bookCard = `<li id=${liId} class= "booklist" >
+      <div class="text">
+      <p>"${book.title}" by</p>
+      <p>${book.author}</p>
```

Comment:
- [ ] [OPTIONAL] The book object is already Destructured in the forEach Loop, now it's time to use the Destructured values instead of using the dot notation:
```JavaScript
`
<p>"${title}" by</p>
<p>${author}</p>
`
```
You can also Destructure all the data you want from an object in one line:
```JavaScript
const { id, title, author } = book;
```